CHANGELOG.md

```diff
@@ -42,7 +44,10 @@ All notable changes to the Wazuh app project will be documented in this file.
 ### Fixed
 
 - Fixed the scripted fields disappear when the fields of the events index pattern was refreshed [#6237](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6237)
+  <<<<<<< HEAD
+- # Fixed an error of malformed tabe row on the generation of PDF reports [#6558](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6558)
 - Fixed the sample alerts scripts to generate valid IP ranges and file hashes [#6667](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6667)
+  > > > > > > > 16d03fb8ab4633f3f9955bd295636ba6f85fa45c
```

Comment:
There seem to be some unresolved conflicts in the changelog.

Reply:
Fixed here https://github.com/wazuh/wazuh-dashboard-plugins/pull/6558/commits/02387c47148deba990a6bdf2219a0a2c4f0d0d18.